app/scripts/Xprize-KynDynRetargeting.xml

```diff
@@ -41,12 +41,21 @@
     </dependencies>
   </module-->
 
-   <module>
-     <name>XsensRetargetingModule</name>
-     <parameters>--from iFeelRetargetingWalking.ini</parameters>
-     <node>icub-console</node>
-   </module>
+  <module>
+    <name>matlab</name>
+    <parameters>-softwareopengl -batch "addpath(genpath(getenv('IFEEL_WALKING_RETARGETING_FOLDER'))); ifeel_walking_retargeting"</parameters>         <!-- parameters -->
+    <node>icub-console-gui</node>
+    <dependencies>
+      <port timeout="15.0">/Wearable/OpenXRTransform/data:o</port>
+      <port timeout="15.0">/iFeelSuit/WearableData/data:o</port>
+    </dependencies>
+  </module>
```

Comment:
I think this is not needed anymore

Reply:
Done in [dd0946a](https://github.com/robotology/walking-teleoperation/pull/108/commits/dd0946a224600979e305aa1bbf2191b766234dc9)